test/js/pageobjects/CrossDeviceConnectedToMobile.js

```diff
@@ -0,0 +1,31 @@
+import Base from './BasePage.js'
+import {locale, verifyElementCopy} from '../utils/mochaw'
+
+class CrossDeviceConnectedToMobile extends Base{
+
+  get connectedToYourMobileTitle() { return this.$('.onfido-sdk-ui-PageTitle-titleSpan')}
+  get connectedToYourMobileSubtitle() { return this.$('.onfido-sdk-ui-PageTitle-titleWrapper > div:nth-child(2)')}
+  get connectedToYourMobileIcon() { return this.$('.onfido-sdk-ui-Theme-icon')}
+  get connectedToYourMobileTips() { return this.$('.onfido-sdk-ui-Theme-header')}
+  get connectedToYourMobileTip1() { return this.$('li:nth-child(1)')}
+  get connectedToYourMobileTip2() { return this.$('li:nth-child(2)')}
+  get connectedToYourMobileTip3() { return this.$('li:nth-child(3)')}
+  get connectedToYourMobileCancel() { return this.$('.onfido-sdk-ui-crossDevice-MobileConnected-cancel')}
```

Comment:
Same here, with the checking for all of the mobile tips list items' text in lines 24-26.

Reply:
Yes, I agree with @kopijunkie. You can create a helper function to iterate over list items

Reply:
Done.